src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -94,13 +100,14 @@
 
 	@Nullable
 	static final MethodHandle TITLE_METHOD;
+	static final boolean HAS_GAMERULES;
 
 	static {
+		HAS_GAMERULES = Skript.classExists("org.bukkit.GameRule");
 		register(ExprName.class, String.class, "(1¦name[s]|2¦(display|nick|chat|custom)[ ]name[s])", "players/entities/itemtypes/inventories/slots");

```

Comment:
Does this work with ExprName properly? You should need to add game rules to the accepted values (unless it already works?)

Reply:
Fixed 👍